### DISCUSSION TITLE
Which Windows version is recommended?

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#420] [Q&A] [answered]
Is this tool more effective or more capable on some Windows versions more than others? For example what would the difference in  effectiveness be between Windows Home and Windows Enterprise? AFAIK Windows Home doesn't have Group Policies so how would something like the Microsoft Security Baselines be applied?

**Comment by HotCakeX** [comment, marked as answer]
Hello,
There are multiple tools in this repository. If you're referring to the Harden Windows Security Module, it does require at least Pro edition, because it uses group policies and Microsoft security baselines.

[AppControl Manager](https://github.com/HotCakeX/Harden-Windows-Security/wiki/AppControl-Manager) application should work on Home edition too. The chart [in here](https://learn.microsoft.com/en-us/windows/security/application-security/application-control/app-control-for-business/feature-availability) states that one would need higher than Home edition to use the PowerShell cmdlets but the app doesn't use them, so that means it brings all of the AppControl/WDAC features to home edition.
